tools/profileBuilder/latest.go

```diff
@@ -0,0 +1,178 @@
+package main
+
+// Copyright 2017 Microsoft Corporation
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/marstr/collection"
+)
+
+// LatestStrategy evaluates the Azure-SDK-for-Go repository for the latest available API versions of each service.
+type LatestStrategy struct {
+	Root      string
+	Predicate func(packageName string) bool
+}
+
+// AcceptAll is a predefined value for `LatestStrategy.Predicate` which always returns true.
+func AcceptAll(name string) (result bool) {
+	result = true
+	return
```

Comment:
I would prefer to keep it simple and `return true`

Reply:
I'll make this change because I like you and Joel. 😝

Reply:
Fixed locally

Reply:
=^w^=